setup.py

```diff
@@ -1,6 +1,6 @@
 from setuptools import setup, find_namespace_packages
 
-version = "0.0.3"
```

Comment:
why bump down the version ?

Reply:
It was a mistake having it at `0.0.5`, the latest pypi release is `0.0.3`.